src/esp/physics/bullet/BulletRigidObject.h

```diff
@@ -439,6 +439,11 @@ class BulletRigidObject : public BulletBase,
    */
   Magnum::Range3D getCollisionShapeAabb() const override;
 
+  /** @brief Object data: All components of a @ref RigidObjectType::OBJECT are
+   * wrapped into one @ref btRigidBody.
+   */
+  std::unique_ptr<btRigidBody> bObjectRigidBody_;
```

Comment:
Why make this public? Is this a hack?

Reply:
Thanks for pointing this out. Unnecessary change.